pydy/viz/camera.py

```diff
@@ -63,6 +63,10 @@ def __init__(self, *args, **kwargs):
         >>> # initializing with Particle, reference_frame ...
         >>> camera3 = PerspectiveCamera('frame3', I, Pa)
         """
+
+        raise RuntimeWarning("Rotation of Perspective Camera does not work " +
```

Comment:
I will just remove #207 from the milestone after merging this. I still needs to be fixed.